src/data/gravity.graphql

```diff
@@ -1231,6 +1247,19 @@ A partner agreement or errors object
 """
 union PartnerAgreementOrErrorsUnion = Errors | PartnerAgreement
 
+"""
+A location
+"""
+type PartnerLocation {
```

Comment:
I gather this [is a new type](https://github.com/artsy/gravity/pull/14388/files) that came along for the ride with this PR